experimental/generation/generator/packages/library/templates/standard/number.schema

```diff
@@ -2,6 +2,15 @@
     "$schema": "http://json-schema.org/draft-07/schema",
     "type": "number",
     "$generator": {
-        "$ref": "minmax.json"
+        "title": "Number",
+        "description": "Map a LUIS number entity like '1' or '3.2' into a number and units.",
+        "allOf": [
+            {
+                "$ref": "minmax.json"
+            },
+            {
+                "$ref": "array.json"
+            }
```

Comment:
For integer schema, the format is        
                "array": {
                    "$ref": "array.json"
                }
Should this formats for integer and number be the same ?